v0/destinations/singular/data/SINGULARSessionNotificationConfig.json

```diff
@@ -0,0 +1,157 @@
+[
+  {
+    "destKey": "p",
+    "sourceKeys": "context.os.name",
+    "required": true
+  },
+  {
+    "destKey": "i",
+    "sourceKeys": "context.app.namespace",
+    "required": true
+  },
+  {
+    "destKey": "app_v",
+    "sourceKeys": "context.app.version",
+    "required": true
+  },
+  {
+    "destKey": "ip",
+    "sourceKeys": ["context.ip", "request_ip"],
+    "required": true
+  },
+  {
+    "destKey": "ve",
+    "sourceKeys": "context.os.version",
+    "required": true
+  },
+  {
+    "destKey": "mo",
+    "sourceKeys": "context.device.model",
+    "required": true
+  },
+  {
+    "destKey": "ma",
+    "sourceKeys": "context.device.manufacturer",
+    "required": true
+  },
+  {
+    "destKey": "lc",
+    "sourceKeys": "context.locale",
+    "required": true
+  },
+  {
+    "destKey": "install_source",
+    "sourceKeys": "properties.referring_application",
+    "required": true
+  },
+  {
+    "destKey": "install_receipt",
+    "sourceKeys": "properties.install_receipt",
+    "required": true
+  },
+  {
+    "destKey": "asid",
+    "sourceKeys": "properties.asid",
+    "required": true
+  },
+  {
+    "destKey": "openuri",
+    "sourceKeys": "properties.url",
+    "required": true
+  },
+  {
+    "destKey": "dnt",
+    "sourceKeys": "properties.dnt",
+    "required": true,
+    "metadata": {
+      "default": true
+    }
+  },
+  {
+    "destKey": "att_authorization_status",
+    "sourceKeys": "context.device.adTrackingEnabled",
+    "required": true
+  },
+  {
+    "destKey": "sessionNotificationName",
```

Comment:
This is basically the event name for the session call, since, in session call, the default event name is SESSION. If the user passes any event name like "Application Install", we can map this event to sessionNotificationName and pass in the params.
CC: @yashasvibajpai